packages/lit-auth-client/src/lib/providers/EthWalletProvider.ts

```diff
@@ -74,7 +74,7 @@ export default class EthWalletProvider extends BaseProvider {
         version: '1',
         chainId,
         expirationTime: expiration,
-        nonce: this.litNodeClient.latestBlockhash || generateNonce(),
+        nonce: this.litNodeClient.latestBlockhash!,
```

Comment:
can we  first check if this is defined? otherwise the `nonce` will be `undefined`